codalab/worker/local_run/local_run_manager.py

```diff
@@ -41,17 +41,20 @@ def __init__(
         cpuset,  # type: Set[str]
         gpuset,  # type: Set[str]
         work_dir,  # type: str
+        shared_file_system=False,  # type: bool
         docker_runtime=docker_utils.DEFAULT_RUNTIME,  # type: str
         docker_network_prefix='codalab_worker_network',  # type: str
     ):
         self._worker = worker
         self._state_committer = JsonStateCommitter(commit_file)
         self._reader = LocalReader()
         self._docker = docker.from_env()
-        self._bundles_dir = os.path.join(work_dir, LocalRunManager.BUNDLES_DIR_NAME)
-        if not os.path.exists(self._bundles_dir):
-            logger.info('{} doesn\'t exist, creating.'.format(self._bundles_dir))
-            os.makedirs(self._bundles_dir, 0o770)
+        self._shared_file_system = shared_file_system
+        if not shared_file_system:
+            self._bundles_dir = os.path.join(work_dir, LocalRunManager.BUNDLES_DIR_NAME)
+            if not os.path.exists(self._bundles_dir):
+                logger.info('%s doesn\'t exist, creating.', self._bundles_dir)
```

Comment:
any plans for an ELK-esque logging setup? say if you can make every logging message traceable to a bundle-uuid/worker-id/server-action etc should make debugging production issues much easier

Reply:
This is a great idea, although probably out of scope for this PR. Can you create an issue for this? And then someone can pick it up in the future.